ProcessLib/PhaseField/CreatePhaseFieldProcess.cpp

```diff
@@ -191,10 +192,29 @@ std::unique_ptr<Process> createPhaseFieldProcess(
         std::copy_n(b.data(), b.size(), specific_body_force.data());
     }
 
+    auto const crack_scheme =
+        //! \ogs_file_param{prj__processes__process__PHASE_FIELD__hydro_crack_scheme}
```

Comment:
This keyword seems not be for this PR, but It is OK to keep it.

Reply:
Yes, you are right. This is already for crack propagation but it is used in `Tests/Data/PhaseField/2D_propagating_petsc.prj` and `Tests/Data/PhaseField/2D_static.prj`.